src/Lecture2.hs

```diff
@@ -181,7 +277,11 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing [] = True
+isIncreasing (x : xs)
+    | xs == [] = True
+    | x < head xs = isIncreasing xs
+    | otherwise = False
```

Comment:
That's a correct solution! 👍🏻 
It uses unsafe functions like `head`. But they are safe in this case 🙂 

You can use pattern matching here to write this function in a slightly more elegant way:

```suggestion
isIncreasing [_] = True
isIncreasing (x : y : ys) = x < y && isIncreasing (y : ys)
```

Reply:
Thanks! That's what was in my mind but I couldn't write it correctly.